src/twister2/cmake_filter/cmakecache.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+# vim: set syntax=python ts=4 :
+#
+# Copyright (c) 2018 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import re
```

Comment:
Should we have some comment that this module was copied 1-1 from V1 and needs some refactoring in the future?

Reply:
Added